### PR TITLE
Assessment template tweak for name and date

### PR DESCRIPTION
**Pull request by chalin**

### CODE REVIEW

File: assessments/template.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
 # Assessment template
 
-Prepared by: `Name`
-Date: `Date`
-
+Prepared by: \<name> ([@add-link-to-your-github-id](https://github.com/cncf/techdocs))<br>
+Date: 2021-mm-dd
```
Comment (nate-double-u) [inline assessments/template.md:4]: This close to the end of 2021, maybe we should make this `yyyy-mm-dd`?

**Review by nate-double-u — APPROVED** [review]
One small suggestion inline. /lgtm even if it isn't used 🙂

File: assessments/template.md (hunk 1 above)
Comment (chalin) [reply assessments/template.md:4]: I'd vote to keep this as is and updated it in January. Then it'll be good for a full year! 😄

**Comment by chalin** [comment]
Thanks for the review @nate-double-u!

File: assessments/template.md (hunk 1 above)
Comment (nate-double-u) [reply assessments/template.md:4]: Can we open timed issues? Set an alarm for ourselves? :smile:

**Pull request merged**